package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "contracts",
+    "version": "0.0.1",
+    "workspaces": [
+        "packages/*"
+    ],
+    "private": true,
+    "description": "TheSandbox game smart contracts",
```

Comment:
The Sandbox Game smart contracts

Reply:
I think that it is: TheSandbox or you are talking about the [G]ame ?